sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -321,6 +322,117 @@ public void queryMetrics(String query, Boolean qmEnabled) {
         }
     }
 
+    @Test(groups = {"simple"}, dataProvider = "query", timeOut = TIMEOUT)
```

Comment:
IMO it would be good to also add some test coverage for diagnostics with more elaborate queries liek aggregates/order-by etc.

Reply:
Yes more tests on their way in the next PR